composer.json

```diff
@@ -8,6 +8,7 @@
   "require" : {
     "xp-framework/core": "^10.0 | ^9.0 | ^8.0 | ^7.0",
     "xp-framework/http": "^10.0 | ^9.0 | ^8.0 | ^7.0",
+    "xp-framework/xml": "^10.0 | ^9.0 | ^8.0",
```

Comment:
Don't CAS servers also speak JSON in the meantime?

Reply:
CAS Protocol 2.0 Specification doesn't include JSON: https://apereo.github.io/cas/6.2.x/protocol/CAS-Protocol-V2-Specification.html, while CAS Protocol 3.0 Specification does: https://apereo.github.io/cas/6.2.x/protocol/CAS-Protocol-Specification.html